plugins/main/public/utils/applications.ts

```diff
@@ -702,7 +702,7 @@ export const wazuhPluginSettings = {
   category: 'management',
   id: 'wazuh-plugin-settings',
```

Comment:
**nitpick**: rename to `app-settings`.

Reply:
done [b1a4838](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6086/commits/b1a483869e88f2ab865dfddf90829d9628153202)